OsmIntegrator/Database/Models/DbTile.cs

```diff
@@ -52,8 +55,63 @@ public class DbTile
 
     public List<DbConversation> Conversations { get; set; }
 
-    public List<DbChangeReport> ChangeReports { get; set; }
+    public List<DbChangeReport> ChangeReports { get; set; } = new();
 
+    public List<DbTileExportReport> ExportReports { get; set; } = new();
+
+    public DateTime? LastExportAt => ExportReports.Any() ? ExportReports.Select(r => r.CreatedAt).Max() : null;
+
+    public string GetUnexportedChanges(IStringLocalizer _localizer)
```

Comment:
I don't think it's a good place for this method ;).

Reply:
discussed